### PR TITLE
[luci-compute] Introduce initial FullyConnected

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce initial compute code for FullyConnected.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #12100 , from  #12099

File: compiler/luci-compute/include/luci_compute/FullyConnected.h [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  FullyConnectedParams _params;
+  // new param from tflite version 5
+  bool _keep_num_dims = false;
+  // shape and data for inputs
+  loco::TensorShape _input_shape;
+  loco::TensorShape _weights_shape;
+  loco::TensorShape _bias_shape;
+  const float *_input_data = nullptr;
+  const float *_weights_data = nullptr;
+  const float *_bias_data = nullptr;
```
Comment (jinevening) [inline compiler/luci-compute/include/luci_compute/FullyConnected.h:76]: Q) Does luci-compute only support fp32 kernel?

Comment (seanshpark) [reply compiler/luci-compute/include/luci_compute/FullyConnected.h:76]: Current luci-interpreter supports U8, S8 and FP32.
This FC implementation comes fro current luci/pass with only FP32.
We may expand with other dtypes later.

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/luci-compute/include/luci_compute/FullyConnected.h (hunk 1 above)
Comment (seanshpark) [reply compiler/luci-compute/include/luci_compute/FullyConnected.h:76]: thanks for the suggestion. I've added https://github.com/Samsung/ONE/issues/10088#issuecomment-1833136497 for future work :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**